src/application/mLoginBox.cpp

```diff
@@ -60,7 +69,7 @@ mLoginBox::mLoginBox(BRect frame)
     errorView->SetFont(&errorFont);
     errorView->SetHighColor(ui_color(B_FAILURE_COLOR));
 
-    BLayoutBuilder::Group<>(this, B_VERTICAL, 0)
+    BLayoutBuilder::Group<>(this, B_VERTICAL, 0 | B_PLAIN_BORDER)
```

Comment:
The third parameter is not for styling, but for spacing: https://www.haiku-os.org/docs/api/classBLayoutBuilder_1_1Group.html#ace860b547d56684445905033b112945e so the B_PLAIN_BORDER flag for box borders will not work as expected here.